code/mob/living/silicon/ai.dm

```diff
@@ -126,6 +126,7 @@ var/global/list/ai_emotions = list("Annoyed" = "ai_annoyed-dol", \
 	var/moustache_mode = 0
 	var/status_message = null
 	var/mob/living/silicon/deployed_shell = null
+	var/locking = 0
```

Comment:
If you're going to mirror this var onto AI frames you should also mirror how it's used - i.e.: resetting to `0` when manually (un)locking, or popping open with tools/etc.

Reply:
Good catch- should be sorted now!